README.md

```diff
@@ -1,27 +1,72 @@
-# Amazon ECS CLI v2
+## Amazon ECS CLI v2 - Develop, Release and Operate Container Apps on AWS
 
 [![Join the chat at https://gitter.im/aws/amazon-ecs-cli-v2](https://badges.gitter.im/aws/amazon-ecs-cli-v2.svg)](https://gitter.im/aws/amazon-ecs-cli-v2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
->⚠️ Before you get started please note that this feature is in preview and the intention is to ensure it meets your requirements and give us any feedback on your use case. Please do not run production workloads till we announce the general availability of this feature. Using the instructions and assets in this repository folder is governed as a preview program under the AWS Service Terms: https://aws.amazon.com/service-terms/ . 
+>⚠️ Before you get started please note that this feature is in preview and the intention is to ensure it meets your requirements and give us any feedback on your use case. Please do not run production workloads till we announce the general availability of this feature. Using the instructions and assets in this repository folder is governed as a preview program under the AWS Service Terms: https://aws.amazon.com/service-terms/ .
 
 The ECS CLI is a tool for developers to create, release and manage production ready containerized applications on ECS.
-From getting started, pushing to staging and releasing to production, the ECS CLI can help manage the entire lifecycle
-of your application development.
+From getting started, pushing to a test environment and releasing to production, the ECS CLI helps you through the entire life of your app development.
 
-Once you've built something you're excited to deploy, let the ECS CLI set up a CI/CD pipeline for you,
-with built in testing hooks and manual gates.
-Tail your logs, monitor your key metrics and push updates all from the comfort of your terminal.
+Got a Dockerfile and some code? Get it up and running on ECS in under 10 minutes, with just one command. Ready to take that app to production? Spin up new environments and a continuous delivery pipeline without having to leave your terminal. Find a bug? Tail your logs and deploy with one tool.
```

Comment:
I like that one sentence summary above, do you think we should have this text under:
`## What's the ECS CLI?"` similar to https://github.com/firecracker-microvm/firecracker